.gitignore

```diff
@@ -11,3 +11,4 @@ PHP_BrowscapINI
 ergebnis.txt
 tests/data/*
 !tests/data/.gitkeep
+.php_cs.cache
```

Comment:
Yeah, pros and cons to that. We commit it in our projects which can cause merge conflicts that can really only be resolved by blowing away the file and re-running php-cs-fixer.  But I believe when the PHP version is different the cache is invalidated, so unless all developers are using same PHP version, doesn't seem to make much sense to commit it.